### DISCUSSION TITLE
using uWebSockets library

### DISCUSSION THREAD

**Discussion by dtau00** [appy-one/acebase#85] [Ideas]
Somewhere down the road when we are trying to extract more performance, we may consider the uWebSockets library

https://github.com/uNetworking/uWebSockets.js/

supposedly ~10x faster than any other node websocket library since it bypasses node, written directly in c++ to the v8 engine.  I've used it before  without issue (well, except it not being in npm).

**Comment by appy-one** [comment]
I agree, I already use it in [acebase-ipc-server](https://github.com/appy-one/acebase-ipc-server)
